mhep/mhep/assessments/tests/views/test_list_assessments_for_organisations.py

```diff
@@ -0,0 +1,75 @@
+from freezegun import freeze_time
+
+from rest_framework.test import APITestCase
+from rest_framework import status
+
+from mhep.users.tests.factories import UserFactory
+from mhep.assessments.tests.factories import AssessmentFactory, OrganisationFactory
+
+
+class TestListAssessmentsForOrganisation(APITestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.organisation = OrganisationFactory.create()
+        cls.org_member = UserFactory.create()
+        cls.organisation.members.add(cls.org_member)
+        super().setUpClass()
+
+    def test_returns_all_assessments_connected_to_organisation(self):
+        AssessmentFactory.create(organisation=self.organisation)
+        AssessmentFactory.create(organisation=self.organisation)
```

Comment:
To test it's actually filtering assessments based on organisaiton, create another assessment in a different (or without) organisation